helix-view/src/document.rs

```diff
@@ -844,6 +844,19 @@ impl Document {
             .map(|language| language.scope.as_str())
     }
 
+    /// Language ID for the document. Either the `language-id` from the
+    // `language-server` configuration, or the document language if no
+    /// `language-id` has been specified.
```

Comment:
Missing `/`?

Reply:
Fixed in https://github.com/helix-editor/helix/pull/1466/commits/2e0d66c394e704ddff4aa9a903290dff46510169.